code/modules/cargo/exports/plantmed.dm

```diff
@@ -0,0 +1,20 @@
+/datum/export/plantmedicine
+        cost = 2 //Gets multiplied based on healing chems inside of seed
+        unit_name = Medicine
+        export_types = list(/obj/item/seeds)
+        SSshuttle.points = SSshuttle.points
+
+/datum/export/plantmed/get_cost(obj/O)
+        var/obj/item/seeds/S = O
+        if S.has_reagent("omnizine, 1)
+                S.reagents.get_reagent_amount("omnizine")
+                        return ..() * cost
+                
+        if S.has_reagent("earthsblood, 1)
+                S.reagents.get_reagent_amount("earthsblood")
+                        return ..() * cost
+                
+/datum/export/plantmed/sell_object(obj/O)
+        var/cost = ..()
+        if SSshuttle.points = 14000
+                return 0
```

Comment:
Not really sure what this does. But this isn't a proper if statement.

should probably be:

if (SSshuttle.points >= 14000)
    return 0

As  a single = sign SETS a variable. Two == compares it. And comparing to see if the shuttle points are exactly 14000 is a really edge case.